lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1509,8 +1504,11 @@ TypeSystemSwiftTypeRef::GetNameImporter() const {
 }
 
 llvm::Triple TypeSystemSwiftTypeRef::GetTriple() const {
-  if (m_module)
-    return m_module->GetArchitecture().GetTriple();
+  if (auto *module = GetModule())
+    return module->GetArchitecture().GetTriple();
+  else if (auto target_sp = GetTargetWP().lock())
+    return target_sp->GetArchitecture().GetTriple();
+  llvm_unreachable("Expected module or target");
```

Comment:
This should be reachable if you run type system queries without a target. There is nothing in the SBAPI that prevents someone from doing this, so we shouldn't crash. Maybe lldb_assert() or logging?

Reply:
thanks. It's now changed to a log.